src/Traveler.js

```diff
@@ -1,9 +1,28 @@
+// const moment = require('moment')
+import moment from 'moment';
+
+
 class Traveler {
-  constructor(id, name, travelerType) {
-    this.id = id;
-    this.name = name;
-    this.travelerType = travelerType;
+  constructor(traveler, trips) {
+    this.id = traveler.id;
+    this.name = traveler.name;
+    this.travelerType = traveler.travelerType;
+    this.allTrips = trips;
+    this.travelersTrips = [];
+  }
+
+  findTrips() {
+    this.travelersTrips = this.allTrips.filter(trip => trip.userID === this.id)
+  };
+
+  pastTrips() {
+    let today = moment(Date.now())
```

Comment:
After (minimal) research, I think moment() will return today. Not a hundo on that though. Food for thought.